components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/OpenIDConnectClaimFilterImpl.java

```diff
@@ -169,6 +173,15 @@ public List<String> getClaimsFilteredByOIDCScopes(Set<String> requestedScopes, S
                         "getting claims for the requested scopes. Therefore returning an empty claim list.");
             }
         }
+        if (LoggerUtils.isDiagnosticLogsEnabled()) {
+            DiagnosticLog.DiagnosticLogBuilder diagnosticLogBuilder = new DiagnosticLog.DiagnosticLogBuilder(
+                    OAuthConstants.LogConstants.OAUTH_INBOUND_SERVICE, "issue-access-token");
```

Comment:
can't we use the action IDs that we have defined above here?

Reply:
Fixed with 0de4707a706d81132f9bcc03408525caf9ca4be1